README.md

```diff
@@ -1,6 +1,7 @@
 # Amazon Pay Module
```

Comment:
AmazonPay

Please use the proper module names here. See also e.g. https://github.com/spryker/event-behavior and others.

Reply:
Fixed.